sunpy/net/dataretriever/sources/secchi.py

```diff
@@ -0,0 +1,72 @@
+import socket
+from datetime import datetime
+
+from sunpy.extern.parse import parse
+from sunpy.net.dataretriever import GenericClient, QueryResponse
+from sunpy.util.scraper import Scraper
+from sunpy.time import TimeRange, parse_time
+
+
+__all__ = ['SECCHIlient']
+
+
+class SECCHIlient(GenericClient):
+    """
+    Provides access to the STEREO/SECCHI data.
+
+    Uses this `archive <https://stereo-ssc.nascom.nasa.gov/pub/beacon/ahead/secchi/img>`.
+    """
+
+    patterns = {
+        'euvi': '{}/img/euvi/{year:4d}{month:2d}{day:2d}/{year:4d}{month:2d}{day:2d}_{}',
```

Comment:
can these "euvi", "cor2" etc be added as 'instr' sources or something and then added to the pattern such that 
```suggestion
        'euvi': '{}/img/{instr}/{year:4d}{month:2d}{day:2d}/{year:4d}{month:2d}{day:2d}_{}',
```